applets/wncklet/workspace-switcher.c

```diff
@@ -147,8 +160,10 @@ static void window_manager_changed(WnckScreen* screen, PagerData* pager)
 
 	if (!wm_name)
 		pager->wm = PAGER_WM_UNKNOWN;
-	else if (strcmp(wm_name, "Metacity (Marco)") == 0)
+	else if (strcmp(wm_name, "Marco") == 0)
```

Comment:
Did you test this? Seems that `marco` still used `Metacity (Marco)` as wm name:
https://github.com/mate-desktop/marco/blob/master/src/core/display.c#L667

Reply:
No, i will do, otherwise i will revert this change.